src/ImageSequencer.js

```diff
@@ -376,6 +376,20 @@ ImageSequencer = function ImageSequencer(options) {
   }
 
 
+  function reorderSteps(source, dest) {
+    var this_ = (this.name == 'ImageSequencer') ? this : this.sequencer;
+    var args = [];
+    var json_q = {};
```

Comment:
I meant this btw, can these be named something better?

Reply:
Are that's what I said in the file these names are used in the other functions as well so it'd be better to change it all together so as to avoid any inconsistency.

Reply:
So you basically don't know what they even mean? :joy:

Reply:
It's fine. APPROVED lol.

Reply:
Hey lets be nice here! I am probably guilty of the ambiguous naming after all :-) so I can take the blame. 

I agree though that let's change it in a consistent manner so as not to confuse folks. And if there are any opportunities to reduce repetitive code, that's great too!

Reply:
Oh sorry. I was just joking anyway, you are lucky that I didn't build sequencer's backend :joy: